### ISSUE TITLE
Changes performed at Tags level are lost if made before the LLM classification

### ISSUE THREAD

**Issue by emilghittasv** [mozilla/sumo#2396]
**Preconditions**
Sign in to SUMO

**Steps to reproduce**
1. Go to https://support.allizom.org/en-US/questions/new/firefox/form
2. Add the following data inside the `Subject` field: `NVDA does not work with Firefox 3938` and the following data inside the `How can we help?` field: `Firefox doesn't work in combination with the NVDA screen reader`
3. Select the `Download and save` option from the `Which topic best describes your question?` and submit the question.
4. Add a couple of tags inside the `Tags` section before the LLM kicks in to perform the re-classification.
5. Wait for the LLM re-classification and reload the page.

**Expected behavior**
The tags added in step 4 are kept even after the LLM re-classification.

**Actual behavior**
If the LLM performs a re-classification, the tags added in step 4 are removed.

**Screencasts**
![Image](https://github.com/user-attachments/assets/9fe50a01-0f52-4e2a-9247-a610631db83f)

**Desktop:**
 - OS: Windows 11
 - Browser: Firefox

**Additional context**
This is similar to https://github.com/mozilla/sumo/issues/2376

**Comment by emilghittasv** [comment]
I can confirm that this issue is verified fixed in `stage`.

I'm moving this ticket inside the `release` column.
